### PR TITLE
added bhattabhi013 profile

### PR DESCRIPTION
**Pull request by bhattabhi013**
## Changes proposed
Added my links and created JSON file for the same.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2416"></a>

### CODE REVIEW

File: data/bhattabhi013.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    "name": "Abhishek Bhatt",
+    "displayStatsPublic": true,
+    "type": "personal",
+    "bio": "Full-Stack developer | Flutter | NodeJs | Angular",
+    "avatar": "https://github.com/bhattabhi013.png",
+    "tags": ["flutter", "nodejs", "angular", "javascript", "typescript", "open-source"],
+    "socials": [
+      { "platform": "twitter", "url": "https://twitter.com/AbhishekBtwt" },
+      { "platform": "github", "url": "https://github.com/bhattabhi013" },
+      { "platform": "instagram", "url": "https://www.instagram.com/bwith_abhi" }
```
Comment (Panquesito7) [inline data/bhattabhi013.json:11]: Is Instagram available in this field? 🤔

File: data/bhattabhi013.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        "url": "https://bhattabhi013.github.io./",
+        "icon": "link"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/AbhishekBtwt",
+        "icon": "twitter"
+      },
+      {
+        "name": "Connect with me on Linkedin",
```
Comment (Panquesito7) [inline data/bhattabhi013.json:30]: Non-blocking: I recommend capitalizing the `i` here. What do you think?

```suggestion
        "name": "Connect with me on LinkedIn",
```

File: data/bhattabhi013.json [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        "icon": "linkedin"
+      }
+    ],
+    "milestones": [
+      {
+        "title": "Vritant App",
+        "date": "Nov 2022",
+        "icon": "globe",
+        "color": "white",
+        "description": "Currenlty working on this amazing app.",
```
Comment (Panquesito7) [inline data/bhattabhi013.json:41]: Non-blocking: You could probably add a bit about your app. 🙂

File: data/bhattabhi013.json (hunk 1 above)
Comment (eddiejaoude) [reply data/bhattabhi013.json:11]: I am not sure, let's deploy and find out 😂  if not we can add it

**Pull request merged**